Bio/Seq.py

```diff
@@ -1391,7 +1394,7 @@ def complement(self):
         return self
 
     def reverse_complement(self):
-        """The reverse complement of an unknown nucleotide equals itself.
+        """Return the reverse complement of an unknown nucleotide equals itself.
```

Comment:
If you think the function does not make sense, then clearly the current docstring is not clear enough. How about:

```python
        def reverse_complement(self):
            """Return the reverse complement of an unknown sequence.

            The reverse complement of an unknown nucleotide equals itself:

            >>> from Bio.Seq import UnknownSeq
            >>> from Bio.Alphabet import generic_dna
            >>> example = UnknownSeq(6, generic_dna)
            >>> print(example)
            NNNNNN
            >>> print(example.reverse_complement())
            NNNNNN
            """
```